tests/cases/fourslash/completionsOverridingMethod13.ts

```diff
@@ -0,0 +1,31 @@
+/// <reference path="fourslash.ts" />
+
+// @Filename: a.ts
+// @newline: LF
+
+////class A {
+////    protected foo(): void {
+////        return;
+////    }
+////}
+////class B extends A {
+////    /**/
+////}
+
+verify.completions({
+    marker: "",
+    isNewIdentifierLocation: true,
+    preferences: {
+        includeCompletionsWithInsertText: true,
+        includeCompletionsWithSnippetText: false,
+        includeCompletionsWithClassMemberSnippets: true,
+    },
+    exact: [
```

Comment:
honestly, I only added that test to showcase that inside a class (the only scenario we're concerned with), we only have to consider the class element keywords versus the class member completions, but I can get rid of the test if it seems superfluous